pkgs/development/python-modules/meshlabxml/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "MeshLabXML";
+  version = "2018.3";
+
+  disabled = pythonOlder "2.7";
```

Comment:
We don't have or will ever have a python older than 2.7.

Reply:
Done.